### PR TITLE
Introduction to Data Integration in Talend Open Studio (TOS)

### LINKED ISSUE #751: Introduction to Data Integration in Talend Open Studio
### Brief Summary: 
Talend Open Studio is an application that provides solutions for big data, data quality, data preparation and data integration. In this application, data from various sources is combined to provide meaningful information. This article provides an overview of data integration in this application.. 

### Key Takeaways: 
1. Introduction to Talend Open Studio
2. What is Talend Data Integration
3. Benefits of Talend Data Integration
4. Components of Talent Data Integration
5. Application of Talend Data Integration

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #751 .

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/introduction-to-data-integration-in-tos/index.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+Data integration developers require open source tools for synchronizing massive data collected from various platforms in the current data-driven world. Talent Open Studio (TOS) is an open-source application that provides user-friendly and cost-effective solutions for data integration. This application is needed in organizations to enhance data accuracy and improved decision making. 
+
+This article provides an overview of data integration in Talend Open Studio. It starts by introducing readers to Talend Open Studio and Talend data integration. The article has provided a step-by-step explanation of how this application can be installed. Various features have been explained to understand the basics of data integration in TOS.
```
Comment (lalith1403) [inline articles/introduction-to-data-integration-in-tos/index.md:3]: This article has provided -> This article provides a step-by-step.

File: articles/introduction-to-data-integration-in-tos/index.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+### Introduction to Talend Open Studio
+Talend Open Studio is an open-source application developed by Talend, a software integration vendor that provides big data, data quality, data preparation, and data integration solutions. 
+
+It is one of the three main products offered by Talend. The other two products include Talend Platforms and Talend Enterprise. Talend Platforms provides solutions such as hybrid cloud and data services. Talend Enterprise provides Enterprise Service Bus (ESB) solutions. 
+
+In Talend Open Studio, data can be combined easily, converted, and updated in different locations. TOS supports data integration, big data, ESB, and ETL (extract, transform, load) implementations. 
+
+Talend Open Studio for data integration is an Eclipse-based tool. In this application, components can be dragged from the palette section of the Graphical User Interface (GUI) and connected to run ETL jobs. This tool enhances the automatic creation of Java codes for jobs. It is connected to various data sources such as Excel, Relational Database Management System (RDBMS), and Dropbox.
```
Comment (lalith1403) [inline articles/introduction-to-data-integration-in-tos/index.md:12]: Link to eclipse.

File: articles/introduction-to-data-integration-in-tos/index.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+Customer A purchases a sim card using a passport or government identification card.
+
+His name is given as KL M
+
+The Address has been given as Houston, Houston.
+
+The phone number has been given as 2815091234
+
+The information collected by all the customers (including Customer A) can be cleaned to provide meaningful information. In this case, Customer A’s new data form will look as follows.
```
Comment (lalith1403) [inline articles/introduction-to-data-integration-in-tos/index.md:31]: collected by -> collected from

File: articles/introduction-to-data-integration-in-tos/index.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+- **Job tab:** This tab provides important information regarding the current job. This includes creation date, creation time, name, and version.
+- **Context tab:** Context variables are set using this tab. A context tab sets different contexts that will be used in the job design.
+- **Component tab:** This tab shows or displays the parameters needed in the configuration of components.
+- **Run tab:** This tab displays logs such as error, start, and end messages. It shows the progress of a job that is being executed.
+  
+### Components of Talend Data Integration
+There are over 800 components in Talend Open Studio. The following are some of the essential elements of Talend data integration.
+- **tFileList:** Folder files are listed by this component with a specific mask pattern. 
+- **tMysqlConnection:** This component provides a connection to Mysql Database, which is an open-source solution for database management.
```
Comment (lalith1403) [inline articles/introduction-to-data-integration-in-tos/index.md:155]: It is MySQL. Change wherever necessary.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Minor changes and this should be good to go! I have a few suggestions. Go through them and make modifications according to what you think is the best way to put forth the subject. Also, please link some major projects/ companies using TOS. When you link it, make sure to describe it in a line or two. 1-2 use cases would be ideal!

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/introduction-to-data-integration-in-tos/index.md (hunk 1 above)
Comment (mbaabuones) [reply articles/introduction-to-data-integration-in-tos/index.md]: Done.

File: articles/introduction-to-data-integration-in-tos/index.md (hunk 2 above)
Comment (mbaabuones) [reply articles/introduction-to-data-integration-in-tos/index.md]: Done.

File: articles/introduction-to-data-integration-in-tos/index.md (hunk 3 above)
Comment (mbaabuones) [reply articles/introduction-to-data-integration-in-tos/index.md]: Done.

File: articles/introduction-to-data-integration-in-tos/index.md (hunk 4 above)
Comment (mbaabuones) [reply articles/introduction-to-data-integration-in-tos/index.md]: Done.

**Comment by mbaabuones** [comment]
@lalith1403 I have finished working on the changes.

**Pull request merged**